internal/githubapi/client.go

```diff
@@ -0,0 +1,30 @@
+package githubapi
+
+import (
+	"net/http"
+
+	"github.com/google/go-github/v44/github"
+	"github.com/shurcooL/githubv4"
+)
+
+type Client struct {
```

Comment:
The consumers are directly using the GitHub clients. This is going to make it harder to unit test. The suggestion would be to use an interface similar to this https://github.com/ossf/scorecard/blob/7ac81a334f7ef5281d3238ffd68499a6a48106a8/clients/repo_client.go#L29 https://github.com/ossf/scorecard/blob/7ac81a334f7ef5281d3238ffd68499a6a48106a8/clients/repo.go#L18

Reply:
Thanks for the feedback, interfaces certainly help with testability and I do need to do some work in this area.

At the moment I can mock the HTTP libraries, but that may not be the most practical approach.

I expect this area to evolve significantly over the next few months.